### ISSUE TITLE
Documentation needs update:  ImportError: cannot import name 'multilevel'

### ISSUE THREAD

**Issue by DSLituiev** [PyWavelets/pywt#141]
From documentation of `waverec`:
```
Examples
--------
>>> from pywt import multilevel
>>> coeffs = multilevel.wavedec([1,2,3,4,5,6,7,8], 'db2', level=2)
>>> multilevel.waverec(coeffs, 'db2')
array([ 1.,  2.,  3.,  4.,  5.,  6.,  7.,  8.])
File:      ~/external/pywt/pywt/_multilevel.py
Type:      function
```

Typing: `from pywt import multilevel` returns ` ImportError: cannot import name 'multilevel'`.

**Comment by grlee77** [comment]
Thanks for letting us know.  I will submit a PR to fix this.  

We recently renamed `multilevel` to `_multilevel`.  We now recommend importing these functions directly from the  top level `pywt` namespace instead.
e.g.
```python
from pywt import wavedec, waverec
```

**Comment by grlee77** [comment]
This was resolved in #142
